.ddev/docker-compose.selenium.yaml

```diff
@@ -19,7 +19,23 @@ services:
       - ddev-global-cache:/mnt/ddev-global-cache
   chrome:
     container_name: ddev-${DDEV_SITENAME}-chrome
-    image: selenium/standalone-chrome:latest
+    image: selenium/standalone-chromium:latest
```

Comment:
This change to the chrome allows ddev to work on arm laptops. selenium/standalone-chrome:latest is not available on arm... whereas selenium/standalone-chrome:4.8 is.... go figure.

Reply:
I suspect you might've already seen it, but [there's this fairly recent post about multi-arch images](https://www.selenium.dev/blog/2024/multi-arch-images-via-docker-selenium/). (Doesn't explain why it worked with 4.8, but it does speak as to why generally the `chrome` image doesn't support arm.)